packages/frontend-core/src/api/user.js

```diff
@@ -234,6 +234,16 @@ export const buildUserEndpoints = API => ({
     })
   },
 
+  /**
+   * Removes multiple user invites from Redis cache
+   */
+  removeUserInvites: async users => {
```

Comment:
Shouldn't the param should be invite ids instead of users?

Reply:
Refactored to pass the invite codes directly instead of user emails.